src/test/java/com/unzer/payment/business/AuthorizationTest.java

```diff
@@ -225,7 +227,7 @@ public void testMarketplaceAuthorize() throws MalformedURLException, HttpCommuni
 		String participantId_2 = MARKETPLACE_PARTICIPANT_ID_2;
 		
 		//create basket
-		Basket maxBasket = getMaxTestBasket();
+		Basket maxBasket = getMaxTestBasketV1();
```

Comment:
The tests shall all use basketV2 except for the tests conducted on Basket v1 class.
Exception: Tests for payment types not yet capable of handling basket v2.